cmd/earthly/main.go

```diff
@@ -287,13 +288,11 @@ func main() {
 	}
 
 	colorMode := conslogging.AutoColor
-	_, forceColor := os.LookupEnv("FORCE_COLOR")
-	if forceColor {
+	if envutil.IsEnvTrue("FORCE_COLOR") {
```

Comment:
This feels stuttery. `envutil.IsTrue` reads decently enough to me. Maybe an even better approach would be something like:

```go
package envutil

type EnvValue string

func (v EnvValue) Truthy() bool {
    switch v {
    case "", "false", "no", "0":
        return false
    default:
        return true
    }
}

func Get(v string) EnvValue {
    return EnvValue(os.GetEnv(v))
}
```

And then we could use it like

```go
if envutil.Get("FORCE_COLOR").Truthy() {
```

Reply:
I like the `envutil.IsTrue` name.

I like that the functional approach is less chars than the OOP version.
```
if envutil.IsTrue("FORCE_COLOR") {
```

vs

```
if envutil.Get("FORCE_COLOR").Truthy() {
```